src/Libraries/CoreNodeModels/CreateList.cs

```diff
@@ -7,17 +7,18 @@
 
 namespace DSCoreNodesUI
 {
-    [NodeName("List.Create")]
-    [NodeDescription("ListCreateDescription", typeof(DSCoreNodesUI.Properties.Resources))]
-    [NodeSearchTags("ListCreateSearchTags", typeof(DSCoreNodesUI.Properties.Resources))]
+    [NodeName("List.ByItems")]
+    [NodeDescription("ListByItemsDescription", typeof(DSCoreNodesUI.Properties.Resources))]
+    [NodeSearchTags("ListByItemsSearchTags", typeof(DSCoreNodesUI.Properties.Resources))]
     [NodeCategory(BuiltinNodeCategories.CORE_LISTS_CREATE)]
     [IsDesignScriptCompatible]
+    [AlsoKnownAs("List.Create")]
     public class CreateList : VariableInputNode
     {
         public CreateList()
         {
-            InPortData.Add(new PortData("index0", Resources.CreateListPortDataIndex0ToolTip));
-            OutPortData.Add(new PortData("list", Resources.CreateListPortDataResultToolTip));
+            InPortData.Add(new PortData("item0", Resources.ByItemsListPortDataIndex0ToolTip));
+            OutPortData.Add(new PortData("list", Resources.ByItemsListPortDataResultToolTip));
```

Comment:
Let's use the new attributes you created here so that we can see the ports in the search tooltip.